kolibri/core/assets/src/api-resources/exam.js

```diff
@@ -7,6 +7,15 @@ class ExamResource extends Resource {
   static idKey() {
     return 'id';
   }
+
+  /**
+   * Returns a Collection of Exams filtered by a collectionId
+   * @param {string} collectionId - e.g. a classId
+   * @returns {Collection<Exam>}
+   */
+  getCollectionForClass(collectionId) {
```

Comment:
I removed this method and put a JSDoc `@example`, but I do think this kind of abstraction and documentation is useful, especially if you're not familiar with how the backend is implemented. When I do a grep for `getCollection`, I see at multiple forms of `getParams` (`undefined`, `{}`, `{ parent: ... }`, `{ member_of: }`, etc.). And it seems that knowing what  the valid args are requires knowing a lot about the underlying Models.